package.json

```diff
@@ -1,4 +1,94 @@
 {
   "name": "builder-react-component",
-  "version": "0.0.0"
+  "version": "0.0.1",
+  "description": "Builder Archetype - React Component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/builder-react-component.git"
+  },
+  "author": "Ryan Roemer <ryan.roemer@formidablelabs.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FormidableLabs/builder-react-component/issues"
+  },
+  "homepage": "https://github.com/FormidableLabs/builder-react-component",
+  "scripts": {
+    "npm:postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || builder run build-lib",
+    "npm:preversion": "builder run check",
+    "npm:version": "builder run clean && builder run build && git add -A dist",
+    "npm:test": "builder run test-frontend",
+    "clean-dist": "rimraf dist",
+    "build-dist-min": "webpack --config node_modules/builder-react-component/config/webpack/webpack.config.js",
```

Comment:
I think a question to consider with all of this is, "how often will people be overriding lots of scripts from the archetypes and their files?" Having worked on something like this and worked with people for a few weeks now that have used it, they're only overriding at an `app` level for `production` optimization. 

Most of the time, what's coming out of the box for dev and CI time are all they want to just get their work done and if they're doing any kind of modification, they should consider:

1. Doing it minimally where it counts (optimizations)
2. Generating their own project archetype (because they're going to probably want to carry that over across all or most of their projects)

Lots of overrides of an archetype defeats the purpose of a tool like this and bogs down a workflow that can be optimized for effectiveness by using `builder`.